src/master.R

```diff
@@ -0,0 +1,49 @@
+### Master file to run all the calculations
+
+# Setting global variables required for the script ####
+Sys.setenv(locale = "en", chrome_download_dir = "/home/olga/Documents/WHO/argus/src/assets")
```

Comment:
I think this shouldn't be set here. The point is to set this outside this code, so that we don't have to modify code itself (we avoid conflicts, accidentally committing credentials to repo etc)

there are many options, like:
1. running `export ARGUS_DASHBOARD_LOCALE="en"` before you run the app
2. adding `export ARGUS_DASHBOARD_LOCALE="en"` stuff to `~/.bashrc` or `~/.zshrc`
3. adding `Sys.setenv` to `~/.Rprofile`
4. adding variables and values (without `export`) before command, like this: `ARGUS_DASHBOARD_LOCALE="en" MY_OTHER_VARIABLE="my value..." Rscript master.R`

Reply:
removed